src/servers/graphql-server.ts

```diff
@@ -82,21 +96,20 @@ const sessionContext = ({
   let domainAccount: Account | undefined
   return addAttributesToCurrentSpanAndPropagate(
     {
-      [SemanticAttributes.ENDUSER_ID]: userId,
+      [SemanticAttributes.ENDUSER_ID]: uid,
       [SemanticAttributes.HTTP_CLIENT_IP]: ip,
     },
     async () => {
-      if (userId) {
+      if (uid) {
+        const userId = uid as unknown as UserId // FIXME: fix until User is attached to kratos
         const loggedInUser = await Users.getUserForLogin({ userId, ip, logger })
         if (loggedInUser instanceof Error)
           throw new ApolloError("Invalid user authentication", "INVALID_AUTHENTICATION", {
             reason: loggedInUser,
           })
         domainUser = loggedInUser
 
-        const loggedInDomainAccount = await Accounts.getAccount(
-          domainUser.defaultAccountId,
-        )
+        const loggedInDomainAccount = await Accounts.getAccount(uid)
```

Comment:
please keep previous code, we have not migrated accounts/users so this works now but will create a problem in the future

Reply:
I mixed uid/aid above. that doesn't have an impact for now. UserId/User Collection (as in: in mongodb) will be removed shortly.

Reply:
let me know if that respond to your concern: 3207ed17bee981bf20b88d9651565707b2033637